LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 GitHub Inc.
```

Comment:
I did a `git log on README` to find the original year was 2011 cc b253e94e051c017eb7c8c0101c30a24f0851a499